korim/src/commonMain/kotlin/com/soywiz/korim/format/TGA.kt

```diff
@@ -1,5 +1,6 @@
 package com.soywiz.korim.format
 
+import com.soywiz.korim.bitmap.Bitmap
```

Comment:
There are now files that have new imports but no other code change. Can you remove those imports so the files are not updated?

Reply:
done